modules/extensions-support/src/main/resources/META-INF/services/org.mule.extension.introspection.declaration.spi.ModelEnricher

```diff
@@ -0,0 +1 @@
+org.mule.module.extension.internal.runtime.connector.ConnectorModelEnricher
```

Comment:
the behavior will we the one of the standard JDK ServiceLoader. The order of the enrichers should not matter. Properly sorting the interceptors is a different story